### PR TITLE
Use generic tag 'screenlock-password' instead of gnome-screenlock-password in ensure_unlocked_desktop

### PR DESCRIPTION
**Pull request by ggardet**
This will fix https://progress.opensuse.org/issues/39734 for LXDE case, once related needles will be merged: https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/420

### CODE REVIEW

File: lib/utils.pm [hunk 1]
```diff
@@ -436,15 +436,15 @@ sub workaround_type_encrypted_passphrase {
 sub ensure_unlocked_desktop {
     my $counter = 10;
     while ($counter--) {
-        assert_screen [qw(displaymanager displaymanager-password-prompt generic-desktop screenlock gnome-screenlock-password)], no_wait => 1;
+        assert_screen [qw(displaymanager displaymanager-password-prompt generic-desktop screenlock gnome-screenlock-password xscreensaver-screenlock-password)], no_wait => 1;
```
Comment (okurz) [inline lib/utils.pm]: How about we replace `gnome-screenlock-password` with `screenlock-password`? I can not imagine a situation where we need to distinguish between the two so it should be easier to reuse the more generic tag

**Comment by jwbruno** [comment]
Hello, it seems that after this PR was merged, QAM tests have started failing in consoletest_finish:
SLE-12-SP3: https://openqa.suse.de/tests/1970532#step/consoletest_finish/15
SLE-12-SP3: https://openqa.suse.de/tests/1973722#step/consoletest_finish/17

**Comment by mitiao** [comment]
So is it missing new needles for sle?

**Comment by okurz** [comment]
Yes, sorry, we missed SLE

**Pull request merged**